sway-lib-std/src/vec.sw

```diff
@@ -208,4 +208,22 @@ impl<T> Vec<T> {
         self.len -= 1;
         Option::Some(read(self.buf.ptr() + self.len * size_of::<T>()))
     }
+
+    /// Swaps two elements
+    /// a - The index of the first element
+    /// b - The index of the second element
+    /// Panics if `a >= self.len || b >= self.len` .
+    pub fn swap(mut self, a: u64, b: u64) {
```

Comment:
> Can you rename `a` and `b` to be a little more meaningful e.g. `element1` and `element2`? I know that rust uses `a` and `b` but that is bad practice and those devs need to get a talking to.

Maybe better will be to rename to `element1_index` and `element2_index`, because `a` and `b` are not elements to swap, but their indexes.
Btw, C++ also uses `a` and `b` naming convention.